API/DTOs/SeriesMetadataDto.cs

```diff
@@ -62,6 +62,10 @@ public class SeriesMetadataDto
     /// A comma-separated list of Urls
     /// </summary>
     public string WebLinks { get; set; }
+    /// <summary>
+    /// Series cover display option
+    /// </summary>
+    public CoverDisplayOption CoverDisplayOption { get; set; }
```

Comment:
I'm not sure this belongs on the Series. 

It is likely this is used for Magazines, which are likely to be in their own library, rather than on a series. Plus if a user has a lot of magazines, it is quite painful to update this for all those series. Maybe @lyingloverr, @MortenCB, or @raub21 can comment on their opinion.